react-carousel/my-app/src/Carousel.tsx

```diff
@@ -0,0 +1,10 @@
+type Prop = {
+  image: {
+    src: string;
+    alt: string;
+  };
+};
+
+export function Carousel({ image }: Prop) {
```

Comment:
Usually the term "Carousel" means the entire component with its rotating images and buttons, not just a single image.